src/components/icon-button/icon-button.pw.tsx

```diff
@@ -0,0 +1,232 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import IconButtonComponent from "./component.test-pw";
+import { CHARACTERS } from "../../../playwright/support/constants";
+import { HooksConfig } from "../../../playwright";
+import { checkAccessibility } from "../../../playwright/support/helper";
+
+test.describe(
+  "check IconButton component focus outlines and border radius",
+  () => {
+    test(`should have the expected styling when the focusRedesignOptOut is false`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<IconButtonComponent aria-label={CHARACTERS.STANDARD} />);
+
+      const iconLocator = page.getByTestId("icon").locator("..");
+      await iconLocator.focus();
+      await expect(iconLocator).toHaveCSS(
+        "box-shadow",
+        "rgb(255, 188, 25) 0px 0px 0px 3px, rgba(0, 0, 0, 0.9) 0px 0px 0px 6px"
+      );
+      await expect(iconLocator).toHaveCSS(
+        "outline",
+        "rgba(0, 0, 0, 0) solid 3px"
+      );
+    });
+
+    test(`should have the expected styling when the focusRedesignOptOut is true`, async ({
+      mount,
+      page,
+    }) => {
+      await mount<HooksConfig>(
+        <IconButtonComponent aria-label={CHARACTERS.STANDARD} />,
+        {
+          hooksConfig: { focusRedesignOptOut: true },
+        }
+      );
+
+      const iconLocator = page.getByTestId("icon").locator("..");
+      await iconLocator.focus();
+      await expect(iconLocator).toHaveCSS(
+        "outline",
+        "rgb(255, 188, 25) solid 3px"
+      );
+    });
+
+    test(`render with the expected border radius`, async ({ mount, page }) => {
+      await mount(<IconButtonComponent />);
+
+      const iconLocator = page.getByTestId("icon").locator("..");
+      await iconLocator.focus();
+      await expect(iconLocator).toHaveCSS("border-radius", "4px");
+    });
+  }
+);
+
+test.describe("check props for IconButton component", () => {
+  test(`should render IconButton with aria-label prop`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<IconButtonComponent aria-label={CHARACTERS.STANDARD} />);
+
+    const iconLocator = page.getByTestId("icon").locator("..");
+
+    await expect(iconLocator).toHaveAttribute(
+      "aria-label",
+      CHARACTERS.STANDARD
+    );
+    await expect(iconLocator).toBeVisible();
+  });
+
+  test(`should render IconButton with children prop`, async ({
```

Comment:
**suggestion** (non-blocking): i know it was written like this in the cypress test, but i'd change this test description. There is no `children` prop and in the test there's only one child. Something like `should render with children` should do